### DISCUSSION TITLE
no variables

### DISCUSSION THREAD

**Discussion by tmanyaa** [blendernc/blendernc#206] [Q&A] [answered]
Hi! im following the import gebco topo tutorial in the blendernc documentation and seem to be unable to select a variable, see attached screenshot- could anyone help? 
![image](https://github.com/user-attachments/assets/47b594ba-1bb8-4664-a10f-e4e870a0fd93)

**Comment by josuemtzmo** [comment, marked as answer]
Hello @tmanyaa, my only guess is that `xarray` is not properly installed alongside Blender.  Can you go to the terminal within the scripting tab in blender and import `xarray`? 

<img width="1800" alt="Screenshot 2024-11-04 at 09 20 00" src="https://github.com/user-attachments/assets/ca6af71e-0626-40ae-b0ee-e032a8df0254">
